### ISSUE TITLE
Connect user and password const?

### ISSUE THREAD

**Issue by EdwinGH** [ChuckBell/MySQL_Connector_Arduino#211]
In my Arduino code I am defining:
`const char* mysql_user= "user";
`
Using const as best practice for defining strings this way (and to prevent compiler warnings like _ISO C++ forbids converting a string constant to 'char*'_).
When calling
`conn.connect(mysql_server, 3306, mysql_user, mysql_password)
`
I get a compiler error:
_error: invalid conversion from 'const char*' to 'char*'_

I looked at the signature of connect:
`boolean MySQL_Connection::connect(IPAddress server, int port, char *user, char *password, char *db)
`
And I wonder why user, password, and db are not defined as const char, as I do not see them changed anywhere.
Any suggestions how to best solve this?

**Comment by ChuckBell** [comment]
Yep. Change it yourself on your PC. :) If it works and you’re happy with
it, you are welcome to make a pill request and submit the change. Behold,
the power of open source!

**Comment by ChuckBell** [comment]
Sorry, I didn’t fully answer your question. I made the original code to an
LCD standard. Most board compilers understand char * but at the time fewer
would permit Cindy char * and there are use cases for not using const.

**Comment by EdwinGH** [comment]
I have made a fork, did the proposed changes, checked that it solves the issue, and submitted the pull request. First time I'm doing this (thanks for encouraging me!), so not sure what happens next. On my side things are working, so happy anyway!
